### PR TITLE
Expand the set of exposed UIViewTester matchers and actions

### PR DESCRIPTION
**Pull request by justinseanmartin**
I found some actions that existed in `KIFUITestActor` that weren't being exposed in `KIFUIViewTestActor`. This tries to bring it to parity. The typing into first responder I solved by introducing `usingFirstResponder` as a matcher.

### CODE REVIEW

**Comment by justinseanmartin** [comment]
@RoyalPineapple @mikelupo - This one too when you get a sec.

**Comment by justinseanmartin** [comment]
Fix for the Travis failure requires rebasing the fix on top of #993 after it is merged.

File: Classes/KIFUIViewTestActor.m [hunk 1]
```diff
@@ -263,9 +283,10 @@ - (void)enterTextIntoCurrentFirstResponder:(NSString *)text fallbackView:(UIView
 - (void)setText:(NSString *)text;
 {
     KIFUIObject *found = [self _predicateSearchWithRequiresMatch:YES mustBeTappable:NO];
```
Comment (mikelupo) [inline Classes/KIFUIViewTestActor.m]: Can *found be nil in this case and if so is it safe to use?
Getting rusty in obj c these days...

Comment (justinseanmartin) [inline Classes/KIFUIViewTestActor.m]: Yea, ObjC doesn't really have any sort of nullability enforcement, it just relies on nil messaging semantics and assumes that you're ok with everything being nullable.

Comment (justinseanmartin) [inline Classes/KIFUIViewTestActor.m]: I had messed up the implementation of `setText:` though, so I just pushed a fix for it.

**Comment by mikelupo** [comment]
Thanks Justin. I see no other issues

**Comment by justinseanmartin** [comment]
Exposed one more action for `KIFUIViewTestActor` that we needed. Also, wrapping all the actions in an `autoreleasepool` to prevent the UIView lifetimes from being extended beyond the lifetime of the action.

**Comment by justinseanmartin** [comment]
Rebased with the changes from #993 now merged.

**Pull request merged**